text/0008-bounded-contexts.md

```diff
@@ -0,0 +1,25 @@
+* Start Date: 11/13/2023
+* Contributors:
+  * [Craig McNally](cmcnally@ebsco.com)
+  * [Vince Bareau](vbareau@ebsco.com)
+* RFC PRs:
+  * PRELIMINARY REVIEW: https://github.com/folio-org/rfcs/pull/20
+  * DRAFT REFINEMENT: TBD
+  * PUBLIC REVIEW: TBD
+  * FINAL REVIEW: TBD
+* Outcome: 
+
+# Bounded contexts
+
+## Summary
+It is proposed that the boundary of the microservices, and thus the bounded context, be extended beyond the module. The new bounded context would be found at the Application level. Thus, any of the backend modules in the Application would be able to directly access the storage layer.
+
+## Scope
+* Cross-module database access 
+* Ensuring data integrity
+* Example use cases
+* Practical considerations for how this will work
+
+## Timing
+* Fourth in the sequence.
```

Comment:
I see that the Application Formalization RFC #14 specifically excludes bounded context (this one) from scope, which is great.  But for the sequence, would it help to work through this one first?

Reply:
Can you please clarify why you see this RFC as more urgent than say Application Management and App-Specific UI bundles?

Reply:
Not "more urgent".  It's just the RFC sequence you have proposed is generally one where each RFC builds upon the prior ones, i.e. application management builds on application formalization.  It seems to me that application formalization builds upon the change in bounded contexts, so it might be easier to consider bounded contexts first.  I don't feel strongly about it, and in any case we approved all four this morning (after your question).